label_sync/main.go

```diff
@@ -647,9 +701,27 @@ type labelData struct {
 
 func writeDocs(template string, output string, config Configuration) error {
 	data := []labelData{
-		{"both issues and PRs", "both-issues-and-prs", config.LabelsByTarget(bothTarget)},
-		{"only issues", "only-issues", config.LabelsByTarget(issueTarget)},
-		{"only PRs", "only-prs", config.LabelsByTarget(prTarget)},
+		{"all repos, for both issues and PRs", "all-repos-for-both-issues-and-prs", LabelsByTarget(config.Default, bothTarget)},
```

Comment:
I am going to add an automatic generation of the link text... as I have just seen org/repo breaks in a web browser. Plus it will make the code easier to maintain.